sdk/typescript/src/cryptography/signature.ts

```diff
@@ -72,24 +70,13 @@ export function parseSerializedSignature(serializedSignature: SerializedSignatur
 				bytes,
 			};
 		case 'ZkLogin':
-			const signatureBytes = bytes.slice(1);
-			const { inputs, maxEpoch, userSignature } = parseZkLoginSignature(signatureBytes);
-			const { issBase64Details, addressSeed } = inputs;
-			const iss = extractClaimValue<string>(issBase64Details, 'iss');
-			const address = computeZkLoginAddressFromSeed(BigInt(addressSeed), iss);
-
+			let res = parseSerializedZkLoginSignature(serializedSignature);
 			return {
-				serializedSignature,
-				signatureScheme,
-				zkLogin: {
-					inputs,
-					maxEpoch,
-					userSignature,
-					iss,
-					address,
-					addressSeed: BigInt(addressSeed),
-				},
-				signature: bytes,
+				serializedSignature: res.serializedSignature,
+				signatureScheme: res.signatureScheme,
+				zkLogin: res.zkLogin,
+				signature: res.signature,
+				publicKey: res.publicKey,
```

Comment:
for some reason if i do 

`return parseSerializedZkLoginSignature(serializedSignature)` here the typescript complains about the type undefined the json. i recasted it here, lmk if this is the right ts approach